src/channels/channelsDialogue.component.test.tsx

```diff
@@ -43,32 +39,60 @@ describe('selectChannelTree', () => {
             ])
           ),
         },
-        test_1: {
-          name: 'test_1',
+        Channels: {
+          name: 'Channels',
           checked: false,
           children: {
-            [testChannels[4].systemName]: {
-              ...testChannels[4],
-              checked: true,
+            '1': {
+              name: '1',
+              checked: false,
+              children: {
+                ...Object.fromEntries(
+                  testChannels
+                    .filter((channel) => channel.path.includes('1'))
+                    .map((channel) => [
+                      channel.systemName,
+                      {
+                        ...channel,
+                        checked:
+                          channel.systemName === 'CHANNEL_ABCDE' ? true : false,
```

Comment:
Minor point, but this can be shortened to `channel.systemName === 'CHANNEL_ABCDE'`.